Update run.md (#2713)
tharsis/testnets: olympus_mons/run.md

systemctl status evmosd
```

> ### Configue Your Firewall

Reviewer: ```suggestion
### Configure Your Firewall (Recommended)
```

> This will contribute to the overall health of the network and is strongly recommended

Reviewer: ```suggestion
This will contribute to the overall health of the network and better secure your node instance.
```

Disable firewall

> `ufw disable`

Reviewer: Can we use code blocks for each segment?

Author: Yes, please make any changes you see fit.  Thanks

Set defaults for incoming/outgoing ports

> `ufw default deny incoming`  
> `ufw default allow outgoing`

Reviewer: ```suggestion
```sh
ufw default deny incoming
ufw default allow outgoing
```
```

Re-enable the firewall

`ufw enable`  

> `ufw status verbose`

Reviewer: can we do a code block
```sh
<instructions>
```

Author: Yes, please make any changes you see fit.  Thanks

Allow inbound peers to connect

```
sudo ufw allow from any to any port 26656 proto tcp
```

> _In production you would configure these firewall settings on your sentries only_

Reviewer: ```suggestion
_In production you would configure these firewall settings on your sentries only. See the [validator security section](https://evmos.dev/guides/validators/security.html) in the Evmos docs for information on making a secure validator set up with sentries._
```
